hazyCalculator.js

```diff
@@ -1,18 +1,31 @@
 function isSkippedValue(value) {
+  if (value === undefined || value === '') {
```

Comment:
This is breaking up the checks which really isn't needed. The issue with `isSkippedValue` is that it worked for everything except `null` because `null` will typecast to `false` which means we would return `true` and treat it as a skipped value. Really we just needed to exclude `null` from this with the addition of `&& value !== null` which is the only exception.